docs/how-to's/ocm-control-plane-walkthrough.md

```diff
@@ -0,0 +1,426 @@
+# Open Cluster Management and Multi-Cluster gateways
```

Comment:
Has this file changed at all, or just moved?

Reply:
just moved